setup.py

```diff
@@ -26,24 +22,21 @@
     os.system('python setup.py sdist upload')
     sys.exit()
 
-packages = [
-    'wsproto',
-]
-
 setup(
     name='wsproto',
     version=version,
     description='WebSockets state-machine based protocol implementation',
     author='Benno Rice',
     author_email='benno@jeamland.net',
-    packages=packages,
+    packages=find_packages(),
```

Comment:
Should probably amend this to exclude `docs` and `tests` from being discovered as "packages".